src/visualisations/gender-disparity/css/annotations.css

```diff
@@ -82,24 +88,37 @@
   background-size: contain;
 }
 
-.female .gap-edge span.annotation::before {
+.employment .female .gap-edge span.annotation::before {
   content: 'Female unemployment rate';
   top: calc(100% + 16rem);
   left: calc(100% + 53rem);
 }
 
+.education .female .gap-edge span.annotation::before {
+  content: 'Percentage of female youth with at least a matric education';
+  top: calc(100% + 16rem);
+  left: calc(100% + 53rem);
+}
+
 .female .gap-edge span.annotation::after {
   top: calc(50% + 5rem);
   left: calc(100% + 10rem);
   transform: rotate(70deg) scale(1, -1);
 }
 
-.male .gap-edge span.annotation::before {
+.employment .male .gap-edge span.annotation::before {
   content: 'Male unemployment rate';
   bottom: calc(100% + 16rem);
   right: calc(100% + 45rem);
 }
 
+.education .male .gap-edge span.annotation::before {
+  content: 'Percentage of male youth with at least a matric education';
```

Comment:
FYI this was bad design in the original - part of the point of CSS is to keep style and content deparate, and now this puts content into CSS so it isn't as reusable as it would otherwise be.

Reply:
Thanks JD, this makes sense! Noted :)